resources/public/api/conf/3.0/retrieve_business_income.yaml

```diff
@@ -1,8 +1,17 @@
 get:
   summary: Retrieve a Business Income Source Summary (BISS)
   description: |
-    This endpoint allows a developer to retrieve a running (year-to-date) summary of the income and expenditure for a specified self-employment,
-    or property business during a tax year. A National Insurance number, business ID, tax year and type of business are required.
+    This endpoint enables you to retrieve a running (year-to-date) summary of the income and expenditure for a 
+    specified self-employment or property business during a tax year. 
+    A National Insurance number, business ID, tax year and type of business are required.
+    
+    <strong>
+      Note: From tax year 2025-26 onwards the following business types are no longer valid:
+        <ul>
+          <li>uk-property-fhl</li>
+          <li>foreign-property-fhl-eea</li>
```

Comment:
This doesn't match the update Sion made. Please update the ordering to match the ordering in description and the enum.